### ISSUE TITLE
[IND_A11y]Unable to dismiss the tooltip of the buttons with escape key. FF issue

### ISSUE THREAD

**Issue by drjoho** [IBMa/equal-access#1096]
## What project/section is the issue in?

- [X ] `a11y Checker`

**IBM Accessibility checker Version 3.1.37**
Keyboard FF

Keyboard checker mode toggle button, Help, Settings button. User should be able to dismiss the tooltip with esc key.

Impact: 
When the   tooltip is not getting dismissed with esc key, keyboard only users may find   difficulty in accessing the application.

WCAG2.1 Mapping: 
1.4.13   Content on Hover or Focus. 

Developer will investigate a fix for FF.

**Comment by drjoho** [comment]
I am not exactly sure what to do here. There are a number of tool tips which behave differently. I need design / SME help to ascertain exactly what they are supposed to do for mouse, keyboard and screen reader.

**Comment by drjoho** [comment]
Experimenting with Tool Tips, mouse, keyboard, and screen reader.

**Comment by drjoho** [comment]
Meeting today to get design and sme guidance.

**Comment by drjoho** [comment]
Meeting resolution:
1. for every interactive element there should be a tooltip if there is NOT a label
2. all tooltips should be able to close with ESC key - works on Chrome not FF

**Comment by drjoho** [comment]
![image](https://user-images.githubusercontent.com/64442976/198074799-6b97ad71-5e86-47ec-af90-dd11de9192f0.png)

**Comment by drjoho** [comment]
Added tooltips to menu next to scan "Scan options". Added tooltip to the switcher, "Focus view". Added keyboard activated tooltips to filters.

**Comment by marcjohlic** [comment]
Fix is primarily for Chrome.   The ESC to close Tool Tip does not close on FF - but we've decided to push the PR through anyway.

**Comment by eirckr** [comment]
escape will not close tooltips in FF only due to the FF env. We will not fix this. Tooltips will work on both FF Chrome.

**Comment by eirckr** [comment]
Validated
